day17/README.md

```diff
@@ -0,0 +1,87 @@
+Question of the day: https://leetcode.com/problems/search-a-2d-matrix/#/description
+
+Write an efficient algorithm that searches for a value
+in an `n` x `m` matrix. This matrix has the following
+properties:
+
+- Integers in each row are sorted from left to right.
+- The first integer of each row is greater than the last
+integer of the previous row.
+
```

Comment:
I think there's an interesting variation to this problem when the matrix is a [Young tableau](https://en.wikipedia.org/wiki/Young_tableau)